boards/innblue/innblue21/doc/index.rst

```diff
@@ -29,7 +29,7 @@ The following parts are built into the board:
 Supported Features
 ==================
 
-The innblue21 board configuration supports the following
+The ``innblue21`` board target supports the following
```

Comment:
here

Reply:
Thanks, I've added this to the opt-out list.